rotkehlchen/db/history_events.py

```diff
@@ -168,6 +170,16 @@ def delete_events_by_tx_hash(
             bindings = [(x,) for x in tx_hashes]
         write_cursor.executemany(querystr, bindings)
 
+    def delete_genesis_tx_events(
+            self,
+            write_cursor: 'DBCursor',
+            address: ChecksumEvmAddress,
+    ) -> None:
+        write_cursor.execute(
```

Comment:
This function is called only in one place for what I'm seeing and the code executed is just a db query. Don't create a new function for this and just move the code to the place where is used